lib/beaker/dsl/install_utils/module_utils.rb

```diff
@@ -133,8 +133,10 @@ def copy_module_to(one_or_more_hosts, opts = {})
               cur_path = File.join(target_module_dir, source_name)
               if (cur_path != target_path)
                 if host.is_powershell?
+                  on host, "rm -Recurse -Force #{target_path}"
```

Comment:
You'll want something cmd friendly here along the lines of:

RMDIR [/S] [/Q] [drive:]path